openmc/mesh.py

```diff
@@ -210,3 +238,99 @@ def get_mesh_xml(self):
             subelement.text = ' '.join(map(str, self._width))
 
         return element
+
+    def build_cells(self, bc=['reflective'] * 6):
+        """Generates a lattice of universes with the same dimensionality
+        as the mesh object in self.  The individual cells/universes produced
```

Comment:
I think this first sentence would be confusing to a Python novice - perhaps re-word as "...with the same dimensionality as the mesh object."